test/testrunner.py

```diff
@@ -316,6 +316,14 @@ def main():
         sm_utils.generate_empty_report(report, test_type, "huggingface")
         return
 
+    # Skipping non AutoGluon specific tests to execute only sagemaker tests
```

Comment:
Seems that this logic is similar to the one above used for `huggingface`. Either configure the above to be used for autogluon or define the common logic in a separate function and call it here.